### ISSUE TITLE
Add More Locale

### ISSUE THREAD

**Issue by Riajyuu** [qzb/dustman#19]
Actually I'm glad to help you with the zh-CN one.
If you'd like to, email me to EMLVIRUS(a)outlook

**Comment by Riajyuu** [comment]
I would be glad to make the zh-CN one. Welcome to email me any moment whenever you need help.
Yep, I'm Chinese.

**Comment by qzb** [comment]
I was thinking about adding locales for some time. It wasn't priority for me because major of Dustman users are using browser with English locale, but If you want to help I will gladly enable it to you 😃 I'll setup support for locales, and let you know when you can start translation. I will also prepare Polish translation by myself and I think I will find someone to do Russian and Spanish ones. I would be nice to have French one too, but I don't know anyone who could do that 🤔

**Comment by Riajyuu** [comment]
@mbid Could u make this ready?
Once your preparation is done, you should expect my PR in few hours.

**Comment by mbid** [comment]
@Rictusempra Yeah, I've made it possible to add translations now. There is already a German one.

Unfortunately, you'll have to edit the html and json manually and create a pull request on Github. I would've added a crowdin.com account to make this more user friendly, but I have to own the Github repo to do that, and it still belongs to @qzb, so I can't.

If that's not a problem, you'll need to provide translation for the files in `_locales/en_US`, `settings/en_US` and `panel/en_US`. Create copies of these folders named according to your locale and translate the files in these directories.
